github/src/main/scala/scabot/github/GithubApi.scala

```diff
@@ -191,6 +193,8 @@ trait GithubApiActions extends GithubJsonProtocol with core.HttpClient {
     def pullRequestCommits(nb: Int)                = p[List[Commit]]      (Get(api("pulls" / nb / "commits")
                                                                                  withQuery Map("per_page" -> "100")))
     def deletePRComment(id: String)                = px                (Delete(api("pulls" / "comments" / id)))
+    def requestReview(nb: Int, reviewers: Reviewers) = px                   (Post(api("pulls" / nb / "requested_reviewers"), reviewers)~>
+                                                                                    addHeader("Accept", "application/vnd.github.black-cat-preview+json"))
```

Comment:
ah, thanks. I'd suggest commenting this and including the link, since otherwise it will be mysterious to any future maintainer